### PR TITLE
Release notes and license PDFs.

### PR DESCRIPTION
**Pull request by awholegunch**
Staged at:

- [Release Notes](http://stg-web101.sw.joyent.continuuity.net/cdap/2.7.0-r2.7.0_doc_finals/en/reference-manual/release-notes.html)
- [Licenses](http://stg-web101.sw.joyent.continuuity.net/cdap/2.7.0-r2.7.0_doc_finals/en/reference-manual/licenses/index.html)

### CODE REVIEW

**Comment by gokulavasan** [comment]
LGTM :+1:

File: cdap-docs/reference-manual/source/release-notes.rst [hunk 1]
```diff
@@ -34,6 +34,80 @@ API Changes
...
+  - Programs are no longer part of a Workflow; instead, they are added in the application
+    and are referenced a Workflow using their names
+  - Schedules are now at the application level and properties can be specified for
+    a Schedules; these properties will be passed to the scheduled program as runtime
+    arguments
+
+Known Issues
+------------
+- See also the *Known Issues* of `version 2.6.1. <#known-issues-261>`_
+- When upgrading an existing CDAP installation to 2.7.0, all metrics are reset.
```
Comment (gokulavasan) [inline cdap-docs/reference-manual/source/release-notes.rst]: @chtyim Is there something we can say about the notification/stream bug here?

Comment (gokulavasan) [inline cdap-docs/reference-manual/source/release-notes.rst]: @awholegunch Can we add this as an known issue:

* Cannot change the TTL of Streams

@chtyim Does this look ok?

**Comment by awholegunch** [comment]
Updated staged versions.

File: cdap-docs/reference-manual/source/licenses/cdap-standalone-dependencies.csv [hunk 2]
```diff
@@ -59,9 +57,11 @@ javax.servlet.javax.servlet-api-3.0.1.jar
 javax.transaction.jta-1.1.jar
 javax.ws.rs.javax.ws.rs-api-2.0.jar
 javax.xml.bind.jaxb-api-2.1.jar
+jdk.tools.jdk.tools-1.7.jar
 jline.jline-2.12.jar
 joda-time.joda-time-2.1.jar
 log4j.log4j-1.2.16.jar
+native
```
Comment (gokulavasan) [inline cdap-docs/reference-manual/source/licenses/cdap-standalone-dependencies.csv]: native?

**Comment by gokulavasan** [comment]
@awholegunch Have you changed other instances of 2.7.0 to 2.7.1?

**Comment by gokulavasan** [comment]
Couple of comments. LGTM otherwise!

File: cdap-docs/reference-manual/source/licenses/cdap-standalone-dependencies.csv (hunk 2 above)
Comment (awholegunch) [inline cdap-docs/reference-manual/source/licenses/cdap-standalone-dependencies.csv]: "native" is one of the dependencies in Standalone dependencies. IIANM, we can ignore it. I will file an issue about changing the license code to ignore it if so. I will check with Sree. It doesn't show up in the previous dependencies; let me see if I can fix it.

**Comment by awholegunch** [comment]
Most other instances of "2.7.0" are generated automatically by the build system.

File: cdap-docs/reference-manual/source/licenses/cdap-standalone-dependencies.csv (hunk 2 above)
Comment (awholegunch) [inline cdap-docs/reference-manual/source/licenses/cdap-standalone-dependencies.csv]: Further investigation: native is being generated by the /native directory in the sdk/lib dir. Inside it is the hadoop.dll native lib.The method of generating the standalone dependencies needs to be updated to include looking in any subdirectories. (I will fix...)

**Comment by awholegunch** [comment]
Reverted back to 2.7.0 in the release notes.

File: cdap-docs/_common/common_conf.py [hunk 3]
```diff
@@ -212,7 +212,7 @@ def print_sdk_version():
   "versions_data":
     { "development": 
         [ ["2.8.0-SNAPSHOT", "2.8.0"], ], 
-      "current": ["2.7.0", "2.7.0"], 
+      "current": ["2.7.1", "2.7.1"],
```
Comment (gokulavasan) [inline cdap-docs/_common/common_conf.py]: I see 2.7.1 here

**Comment by gokulavasan** [comment]
LGTM :+1:

**Pull request merged**